### PR TITLE
ICS Support & Webfont icons

### PR DESCRIPTION
**Pull request by tbartelmess**
- ICS support, for those that are not living in the google world
- Webfont icons, mainly because I didn't have a good bitmap icons for ICS

### CODE REVIEW

File: events/views.py [hunk 1]
```diff
@@ -31,3 +32,11 @@ def get(self, request):
         }
 
         return self.render_to_response(context)
+
+def ics(request, event_id):
+    try:
+        event = Event.objects.get(pk=event_id)
+        return HttpResponse(event.ics_string, content_type='text/calendar')
+    except Event.DoesNotExist:
+        return HttpResponseNotFound()
```
Comment (ashchristopher) [inline events/views.py]: should probably be

```
raise Http404
```

?

File: events/urls.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
 from django.conf.urls import url, patterns
 
-from events.views import EventsListView
+from events.views import EventsListView, ics
 
 
 urlpatterns = patterns('',
     url(r'^$', EventsListView.as_view(), name='events-list'),
+    url(r'^ics/(?P<event_id>\d+)/$', ics)
```
Comment (ashchristopher) [inline events/urls.py]: Add as a named url?

**Comment by ashchristopher** [comment]
PR looks good - mind making those 2 changes? I will try to get it merged and deployed soon.

**Comment by tbartelmess** [comment]
Updated

**Comment by ashchristopher** [comment]
This change got pushed tonight. Just fyi.

**Pull request merged**